src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerTeamOrchestrator.cs

```diff
@@ -198,9 +204,14 @@ public async Task<OrchestratorResponse<AccountDashboardViewModel>> GetAccount(st
                     RequiresAgreementSigning = pendingAgreements.Count(),
                     AgreementsToSign = pendingAgreements.Count() > 0,
                     SignedAgreementCount= agreementsResponse.EmployerAgreements.Count(x => x.HasSignedAgreement),
-                    PendingAgreements = pendingAgreements
+                    PendingAgreements = pendingAgreements,
+                    ApprenticeshipEmployerType = (ApprenticeshipEmployerType)Enum.Parse(typeof(ApprenticeshipEmployerType), apiGetAccountTask.Result.ApprenticeshipEmployerType, true)
                 };
 
+                //note: ApprenticeshipEmployerType is already returned by GetEmployerAccountHashedQuery, but we need to transition to calling the api instead.
```

Comment:
Do we need this comment?  It seems obvious you may want to parse to an enum.

Reply:
Yes, I think it's an important comment. Without it, a developer might notice that `ApprenticeshipEmployerType` is returned by both the query and the api call, and remove the call to the api call as it's effectively redundant (no other property is used from the api call).

However, we've been asked to pick up the flag from the api, as we want to transition to retrieving data through the api, rather than going direct to the database, so it wouldn't be appropriate to remove the api call.